src/Driver/PDO/SQLite/Driver.php

```diff
@@ -45,8 +45,10 @@ public function connect(array $params)
             throw Exception::new($exception);
         }
 
+        /** @var callable(string, callable, int=, int=): bool $callable */
+        $callable = [$pdo, 'sqliteCreateFunction'];
```

Comment:
Why do we need this change? Shouldn't PHPStan and Psalm recognize the type of this callable and if they don't: shouldn't the issue be fixed on their side?

Reply:
I will restore this one.